content/learn/08.contributions/03.arduino-creating-library-guide/arduino-creating-library-guide.md

```diff
@@ -227,4 +255,4 @@ If you'd like to check out the complete library (with keywords and example), you
 
 That's all for now but I'll probably write an advanced library tutorial soon. In the meantime, if you have any problems or suggestions, please post them to the [Software Development forum](https://forum.arduino.cc/c/development/libraries/39).
```

Comment:
```suggestion
If you have any problems or suggestions, please post them to the [Software Development forum](https://forum.arduino.cc/c/development/libraries/39).
```
No point in using up space in the tutorial for speculation about a tutorial that has been hypothetical for a decade.